utils/pretty.py

```diff
@@ -24,13 +24,55 @@ def x(o):
     return x
 
 
+def process_attrs(attrs):
+    """Enables us for more freedom in attribute specification"""
+    if isinstance(attrs, basestring):
+        return [attrs]
+    elif attrs is None:
+        return []
+    elif isinstance(attrs, (list, tuple)):
+        return attrs
```

Comment:
It'd be worth returning `list(attrs)` here to give `process_attrs` a predictable interface, and maybe checking against [collections.Sequence](https://docs.python.org/2/library/collections.html#collections.Sequence) instead of list and tuple explicitly.